homeassistant/components/landisgyr_heat_meter/sensor.py

```diff
@@ -90,19 +97,34 @@ def _handle_coordinator_update(self) -> None:
             else:
                 self._attr_native_value = asdict(self.coordinator.data)[self.key]
 
+        # Some models will supply MWh directly. If not, GJ will be converted to MWh.
         if self.key == "heat_usage":
-            self._attr_native_value = convert_gj_to_mwh(
-                self.coordinator.data.heat_usage_gj
-            )
+            if hasattr(self.coordinator.data, "heat_usage_mwh") and isinstance(
+                self.coordinator.data.heat_usage_mwh, float
+            ):
+                self._attr_native_value = self.coordinator.data.heat_usage_mwh
+            else:
+                self._attr_native_value = convert_gj_to_mwh(
+                    self.coordinator.data.heat_usage_gj
+                )
```

Comment:
We should not convert, instead expose the sensor as GJ

Reply:
Converting has the benefit that the usage can be added to the energy dashboard. The heat ('warmth') is used to warm our houses and is an alternative to gas. 
The conversion also was a topic during the [initial PR](https://github.com/home-assistant/core/pull/73363). As a result also this PR was introduced and merged: https://github.com/home-assistant/core/pull/76016.

Reply:
GJ has been added for the energy dashboard... we should not convert values.

Reply:
Ah I see. Didn't know about that change, but I found [the trigger](https://community.home-assistant.io/t/wth-allow-gj-gigajoule-as-a-long-term-statistic/469968/5). 
I'll remove the conversion. But that will be a breaking change. This PR is only about adding unconverted MWh, which I see as a seperate change. Is it okay to do that change in a seperate PR?

Reply:
I've applied the requested change. I've also made this PR a breaking change.

Reply:
> GJ has been added for the energy dashboard... we should not convert values.

Hi @frenck I've removed the conversion, as requested. If this PR is approved this will help many users move on. If there is still a problem with this PR. please let me know.